TestTables/CMIP7_CV.json

```diff
@@ -0,0 +1,295 @@
+{
+    "CV":{
+        "area_label":[
+            "air",
```

Comment:
I could see utility in converting this into a dictionary, adding the descriptive text
```python
"air":"air",
...
"gis":"grounded_ice_sheet",
...
"lus":"sector (auxiliary coordinate for land-use area types)"
...
```

Doing this across all labels would be useful

Reply:
yes, we'll likely do something like that for all the "simple" CVs, like this one.